### DISCUSSION TITLE
Auxiliary/Extra Files?

### DISCUSSION THREAD

**Discussion by isaac-peter** [sleepymalc/VSCode-LaTeX-Inkscape#34] [Q&A] [answered]
I understand that one should not delete the auxiliary files that are created upon compilation. Any idea how to reduce the clutter that the non .tex or .pdf files cause?

**Comment by sleepymalc** [comment, marked as answer]
I have this in my `setting.json`:
```json
"latex-workshop.latex.clean.fileTypes": [
		"*.aux",
		"*.bbl",
		"*.blg",
		"*.idx",
		"*.ind",
		"*.lof",
		"*.lot",
		"*.out",
		"*.toc",
		"*.acn",
		"*.acr",
		"*.alg",
		"*.glg",
		"*.glo",
		"*.gls",
		"*.fls",
		"*.log",
		"*.fdb_latexmk",
		"*.snm",
		"*.synctex(busy)",
		"*.synctex.gz(busy)"
	],
```

**Reply by isaac-peter** [reply]
Hi Sleepymac! 

Thanks for the blazing fast reply, and your amazing work on this project!

Quick follow-up question for you. Most of the stack exchange articles I've read on this issue say one should NOT delete the auxiliary files. At best, the general guidance is to store them in a different location. Do you have a different perspective on this?

Thanks again

**Reply by sleepymalc** [reply]
Yes, I also saw that before. I just don't like the auxiliary files flying around my local system, so I live with a slightly slower compile time. So it depends. Since I didn't try to adapt this approach (store them in a different directory), so I don't how to set up this in VSCode correctly :(

**Reply by isaac-peter** [reply]
Thanks!!
